youtube_dl/extractor/imgur.py

```diff
@@ -21,12 +21,25 @@ class ImgurIE(InfoExtractor):
             'title': 'MRW gifv is up and running without any bugs',
             'description': 'The Internet\'s visual storytelling community. Explore, share, and discuss the best visual stories the Internet has to offer.',
         },
+    }, {
+        'url': 'https://imgur.com/A61SaA1',
+        'info_dict': {
+            'id': 'A61SaA1',
+            'ext': 'mp4',
+            'title': 'MRW gifv is up and running without any bugs',
+            'description': 'The Internet\'s visual storytelling community. Explore, share, and discuss the best visual stories the Internet has to offer.',
+        },
     }]
 
     def _real_extract(self, url):
         video_id = self._match_id(url)
         webpage = self._download_webpage(url, video_id)
 
+        sources = re.findall(r'<source src="([^"]+)" type="([^"]+)"', webpage)
```

Comment:
This is what happens if you try to download an image from imgur instead of a video:

```
python -m youtube_dl https://imgur.com/hL8R5YG
[Imgur] hL8R5YG: Downloading webpage
WARNING: unable to extract width; please report this issue on http://yt-dl.org/bug
WARNING: unable to extract height; please report this issue on http://yt-dl.org/bug
ERROR: Unable to extract video elements; please report this issue on https://yt-dl.org/bug . Make sure you are using the latest version; see  https://yt-dl.org/update  on how to update. Be sure to call youtube-dl with the --verbose flag and include its complete output.
```